doc/use_cases/navigation_with_cartographer.md

```diff
@@ -0,0 +1,51 @@
+
+**Navigation 2 with Cartographer**
+
+This document explains how to use Navigation 2 with SLAM. The following steps show ROS 2 users how to generate occupancy grid maps and use Navigation 2 to move their robot around.
+
+## Before the Tutorial
+
+- Install ROS 2
+- Install Navigation 2
+- Install SLAM package
+- Install your robot package
+- Source setup.bash in your ROS2 and Navigation 2 installation/build directory or workspaces
+
+## Tutorial Steps
+
+**1- (Robot)**  Bring up the robot-related packages on your robot. 
+
+Make sure /tf and /odom are being published.
+
+- *Ex: Turtlebot 3*
+            
+       - ros2 launch turtlebot3_bringup robot.launch.py
+    
+**2- (Robot or PC)** Bring up Navigation
+
+This will bring up Navigation 2 without nav2_amcl and nav2_map_server. Cartographer will publish to /map topic and do localization.
```

Comment:
We should add a note here about the QOS setting in this case. I actually just noticed that the navigation launch file is no longer defaulting to` map_subscribe_transient_local = false`. Either that needs to be changed back or it needs to be a command line param here:
`ros2 launch nav2_bringup nav2_navigation_launch.py map_subscribe_transient_local:=false`

Reply:
> I have a few suggestions. Also, can a link to this be added to the nav2_bringup README file so that it can be found easily?

Good idea. I will add the links for ROS2  installation, Nav2 build etc. as well.